### PR TITLE
feature: include alternative merging algorithm

### PR DESCRIPTION
**Pull request by welozano**
Implementation of an alternative merging algorithm that allows to find more than one OBA match for each GT data row.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,19 @@ The csv file must include the following columns:
 `--minTripLength 60` will remove, from the oba generated data, activities whose `Origin-Destination Bird-Eye Distance* (meters)` is less than 60 meters.
 * `--tolerance <milliseconds>` Maximum tolerated difference (milliseconds) between matched ground truth data start activity and OBA data start activity. 
 By default, it is 3000 milliseconds. Example usage: `--tolerance 5000` will consider only a difference equal or less than 5000 milliseconds while looking for a match between a ground truth data start activity and a OBA data start activity.
+* `no-iterateOverTol` When used, the merging process will only be applied once over the tolerance defined by `tolerance`.
```
Comment (barbeau) [inline README.md]: Should this and the example have dashes in front? `--no-iterateOverTol`

File: README.md [hunk 2]
```diff
@@ -42,6 +42,19 @@ The csv file must include the following columns:
...
+* `--no-removeStillMode` When used, preprocess of input datasets will not eliminate the records with activity mode equal to `STILL`.
+By default, preprocess of input dataset eliminates the records with activity mode equal to `STILL`.
+Example usage: `--no-removeStillMode`.
+* `--mergeOneToOne` This flag will force the merging system to merge each `Ground Truth trip` with one and only one OBA 
+record according to the other command line parameters. By default, this flag is set to False. In such case, the merger 
+will match each `Ground Truth trip` with all the OBa records that starts after the `Ground Truth trip` starts and before
+the `Ground Truth trip` ends. Example usage: `--mergeOneToOne`
+* `--repeatGtRows` This flag will force the merging system to repeat a GT trip as many rows as matches are found
+before exporting the output. By default, this flag is set to False. In such case, the merger wil only include
+one DT data row per trip while merging with a device. Example usage: `--repeatGtRows`
```
Comment (barbeau) [inline README.md:70]: In the "Run" section, could you update the example to the latest command you're using to match multiple OBA records to a single GT record? In other words, the command that should be run to get the datasets that we've been looking at lately? I would also change the default values to yield those results so as few parameters need to be supplied as possible.

File: README.md [hunk 3]
```diff
@@ -42,6 +42,19 @@ The csv file must include the following columns:
 `--minTripLength 60` will remove, from the oba generated data, activities whose `Origin-Destination Bird-Eye Distance* (meters)` is less than 60 meters.
 * `--tolerance <milliseconds>` Maximum tolerated difference (milliseconds) between matched ground truth data start activity and OBA data start activity. 
 By default, it is 3000 milliseconds. Example usage: `--tolerance 5000` will consider only a difference equal or less than 5000 milliseconds while looking for a match between a ground truth data start activity and a OBA data start activity.
+* `no-iterateOverTol` When used, the merging process will only be applied once over the tolerance defined by `tolerance`. 
+By default, the merging process is applied over tolerances iterating from 30000 to `tolerance` in steps of 30000.
+Example usage: `no-iterateOverTol`.
+* `--no-removeStillMode` When used, preprocess of input datasets will not eliminate the records with activity mode equal to `STILL`.
```
Comment (barbeau) [inline README.md:48]: Could you specify if this applies to GT or OBA records or both? Same for any other parameters where that's not clear.

File: README.md (hunk 2 above)
Comment (barbeau) [reply README.md:70]: Could you also provide example output data for the latest dataset, similar to the example input Markdown table above?

**Review by barbeau — COMMENTED** [review]
Thanks @welozano! This looks good - a few comments on the documentation.

File: README.md (hunk 3 above)
Comment (welozano) [reply README.md:48]: I corrected the arguments and documentation. I will generate a MD table similar to the CSV file with dummy data to upload as example.

File: README.md (hunk 2 above)
Comment (welozano) [reply README.md:70]: I just added this change in the last commit.

**Review by barbeau — APPROVED** [review]
👍  LGTM! Please feel free to merge when ready.

**Pull request merged**